docs/OVTF_integration/OVTF_integration.md

```diff
@@ -0,0 +1,123 @@
+
+For more details, please refer to the [installation](ovtf_install.md) and [build from source](ovtf_build.md) guides.
+
+### Configuration
+
+Once you've installed **OpenVINO™ integration with TensorFlow**, you can use TensorFlow to run inference using a trained model.
+
+For the best results, it is advised to enable [oneDNN Deep Neural Network Library (oneDNN)](https://github.com/oneapi-src/oneDNN) by setting the environment variable of `TF_ENABLE_ONEDNN_OPTS=1`.
+
+To see if **OpenVINO™ integration with TensorFlow** is properly installed, run:
+
+    python3 -c "import tensorflow as tf; print('TensorFlow version: ',tf.__version__);\
+                import openvino_tensorflow; print(openvino_tensorflow.__version__)"
+
+This should produce an output like:
+
+        TensorFlow version:  2.7.0
+        OpenVINO integration with TensorFlow version: b'1.1.0'
+        OpenVINO version used for this build: b'2021.4.2'
+        TensorFlow version used for this build: v2.7.0
+        CXX11_ABI flag used for this build: 0
+
+By default, Intel® CPU is used to run inference. However, you can change the default option to either Intel® integrated GPU or Intel® VPU for AI inferencing. Invoke the following function to change the hardware on which inferencing is done.
+
+    openvino_tensorflow.set_backend('<backend_name>')
+
+Supported backends include 'CPU', 'GPU', 'GPU_FP16', 'MYRIAD', and 'VAD-M'.
+
+To determine what processing units are available on your system for inference, use the following function:
+
+    openvino_tensorflow.list_backends()
+    
+For more API calls and environment variables, see [USAGE.md](docs/USAGE.md).
+
+**Note:** If a CUDA-capable device is present in the system then set the environment variable CUDA_VISIBLE_DEVICES to -1. 
+
+## Examples
+
+To see what you can do with **OpenVINO™ integration with TensorFlow**, explore the demos located in the [examples folder](https://github.com/openvinotoolkit/openvino_tensorflow/tree/master/examples) in our GitHub repository.  
+
+## Try it on Intel® DevCloud
+Sample tutorials are also hosted on [Intel<sup>®</sup> DevCloud](https://www.intel.com/content/www/us/en/developer/tools/devcloud/edge/build/ovtfoverview.html). The demo applications are implemented using Jupyter Notebooks. You can interactively execute them on Intel® DevCloud nodes, compare the results of **OpenVINO™ integration with TensorFlow**, native TensorFlow and OpenVINO™. 
+
+## License
+**OpenVINO™ integration with TensorFlow** is licensed under [Apache License Version 2.0](LICENSE).
+By contributing to the project, you agree to the license and copyright terms therein
+and release your contribution under these terms.
+
+## Support
+
+Submit your questions, feature requests and bug reports via [GitHub issues](https://github.com/openvinotoolkit/openvino_tensorflow/issues).
+
+## How to Contribute
+
+We welcome community contributions to **OpenVINO™ integration with TensorFlow**. If you have an idea for improvement:
```

Comment:
why is it a part of OpenVINO official documentation? It should in readme on github !